server/config/base/header_v2.json

```diff
@@ -0,0 +1,203 @@
+[
```

Comment:
We'll have to think about a good way to add translations to these strings (in a follow-up pr)